pkg/apis/triggers/config/feature_flags.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2020 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	StableAPIFields        = "stable"
+	AlphaAPIFields         = "alpha"
+	enableAPIFields        = "enable-api-fields"
```

Comment:
bikeshed: Allow the same lines as above, I'd avoid using a plural key for something that can only have 1 value.

Reply:
renamed to `enableAPIFieldsKey`. We can use the Key prefix for all future feature flag key names